frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -554,20 +600,17 @@ function ReaderHighlight:onTap(_, ges)
 end
 
 function ReaderHighlight:onTapPageSavedHighlight(ges)
-    local pages = self.view:getCurrentPageList()
+    -- local pages = self.view:getCurrentPageList()
```

Comment:
OK, so this function is called only with non-cre documents.
The book about economics is an EPUB - so we should check how iterating all bookmarks - and calling `getPageBoxesFromPositions()` & co on all of them - is costly, with a PDF book with many bookmarks (I have none).
May be we should filter them by page being in this getCurrentPageList() before going with the more expensive stuff?